microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -23,36 +19,6 @@ On Windows 11, widgets appear in the widgets board which you open from the left
 In Windows 11, Progressive Web Apps (PWAs) can define widgets, update them, and handle user interactions within them.
 
 
-<!-- ====================================================================== -->
-## Enable widgets in Microsoft Edge
-
-Widgets are experimentally supported starting with Microsoft Edge 108.
-
-#### Enable local development of widgets
-
-To enable developing widgets on your local machine:
-
-1. Install [WinAppSDK 1.2](/windows/apps/windows-app-sdk/older-downloads#windows-app-sdk-12).
```

Comment:
I believe installign WinAppSDK 1.2 is still required for local development, but the steps below are no longer necesary

Reply:
i plan to create Chat thread for this PR, & plan to restore this line

Reply:
Restored heading "Enable local development of widgets" promoted to h2.  
Restored the single step.
In Demo App section, restored/reworded step 1:
```
1. If not done yet, install [WinAppSDK 1.2](/windows/apps/windows-app-sdk/older-downloads#windows-app-sdk-12).
```
@captainbrosset @amandabaker